great-idea/css/services.css

```
@@ -0,0 +1,186 @@
+/* http://meyerweb.com/eric/tools/css/reset/ 
+   v2.0 | 20110126
+   License: none (public domain)
+*/
+
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed, 
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+/* Your code starts here! */
+
+h1{
+    font-size: 25px;
+}
+
+.master{
+	display:flex;
+	width: 100%;
+	justify-content: center;
+}
+
+.container{
+	display:flex;
+	flex-direction: column;
+	justify-items: center;
+	width: 876px;
+}
+header{
+	display: flex;
+	justify-content: space-between;
+	width:100%;
+	height: 10%;
+	align-items: center;
+    padding-top: 25px;
+}
+
+.services{
+    display: flex;
+    flex-direction: column;
+    margin-top: 3%;
+    border-bottom: 1px solid black;
+    height: 8%;
+}
+
+nav{
+	display: flex;
+	align-content: center;
+	justify-content: space-around;
+    align-content: baseline;
+    width: 70%;
+}
+
+nav a{
+    text-decoration: none;
+    color: #A7A7A7;
+    font-weight: 600;
+}
+
+.grey-box-container{
+    display:flex;
+    flex-wrap: wrap;
+    width:100%;
+    height: 900px;
+    align-content: space-between;
+    justify-content: space-between;
+    margin-top: 5%;
+    padding-bottom: 5%;
+    border-bottom: 2px solid black;
+}
+
+.grey-box{
+    border:1px solid black;
+    display:flex;
+    align-content: space-evenly;
+    justify-content: space-evenly;
+    background-color: #F4F3F4;
+    padding: 10px;
+    flex-direction: column;
+    width: 48%;
+    height: 30%;   
+}
+
+.button{
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    margin-top: 15px;
+    background-color: rgb(255, 254, 255);
+    border: 1px solid black;
+    height: 15%;
+    width: 35%;
+    border-radius: 5px;
+}
+
```

```suggestion
.button:hover {
/*some styles*/
}
```
Adding a simple hover effect can make the site look a lot better